tests/Aspire.Cli.Tests/Projects/ProjectUpdaterTests.cs

```diff
@@ -38,7 +38,7 @@ await File.WriteAllTextAsync(
             appHostProjectFile.FullName,
             $$"""
             <Project Sdk="Microsoft.NET.Sdk">
-                <Sdk Name="Aspire.AppHost.Sdk" Version="9.5.1-preview.1" />
```

Comment:
Revert these tests, they should be self contained and the concrete version should not affect the outcome.